protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -849,6 +849,8 @@ func schemaOfField(f *descriptor.Field, reg *descriptor.Registry, refs refMap) o
 		}
 	}
 
+	ret.Required = slices.Compact(ret.Required)
```

Comment:
Will the fields in `ret.Required` be sorted already? `slices.Compact` removes _consecutive runs of the same element_ so for deduplication we will need to either guarantee that they are sorted already, or sort first.

Reply:
If that's the case, would the previous version of uniqArray be a better implementation?

Reply:
No, as I said, just sort it first
```suggestion
	slices.Sort(ret.Required)
	ret.Required = slices.Compact(ret.Required)
```